infra/feast-operator/cmd/main.go

```diff
@@ -166,6 +167,22 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "FeatureStore")
 		os.Exit(1)
 	}
+
+	// Setup Notebook ConfigMap controller for OpenDataHub integration
+	// Default to kubeflow.org/v1 Notebook CRD (used by OpenDataHub)
```

Comment:
I think we should probably check for notebooks CRD is installed in cluster as a safe check ?

Reply:
If the reconciliation run due to feast CR updates, then until controller does not find any notebook with project annotation it wont run reconcilation. So this anyways wont trigger. 

Considering the changes are going into ODH and not feast upstream, where ntoebook is set as Managed component. Do you still think we should validate the notebook CRD availability explicitely ?

Reply:
There would be graceful messages anyways and would be the case even if we check notebook CRD availability. Do you think we should hold reconcilation completely when no notebook CRD ?